bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ReadRowsRequestManager.java

```diff
@@ -17,47 +17,27 @@
 
 import static com.google.common.base.Preconditions.checkArgument;
 
-import com.google.api.client.util.BackOff;
-import com.google.api.client.util.Sleeper;
 import com.google.bigtable.v2.ReadRowsRequest;
 import com.google.bigtable.v2.RowRange;
 import com.google.bigtable.v2.RowRange.EndKeyCase;
 import com.google.bigtable.v2.RowRange.StartKeyCase;
 import com.google.bigtable.v2.RowSet;
-import com.google.cloud.bigtable.config.Logger;
-import com.google.cloud.bigtable.config.RetryOptions;
-import com.google.cloud.bigtable.grpc.async.BigtableAsyncRpc.RpcMetrics;
-import com.google.cloud.bigtable.grpc.io.IOExceptionWithStatus;
 import com.google.protobuf.ByteString;
 
-import io.grpc.Status;
-
-import java.io.IOException;
 import java.nio.ByteBuffer;
-import java.util.concurrent.atomic.AtomicInteger;
 
 /**
  * Keeps track of Rows returned from a readRows RPC for information relevant to resuming the RPC
- * after temporary problems like DEADLINE_EXCEEDED.
+ * after temporary problems.
  *
  * @author sduskis
  * @version $Id: $Id
  */
-public class ReadRowsRequestRetryHandler {
+public class ReadRowsRequestManager {
```

Comment:
Might be a good idea to make this class package protected to signify that its not part of the public api